tasks/main.yml

```diff
@@ -6,6 +6,14 @@
   tags:
     - install_apps
 
+# Source: https://www.cyberciti.biz/faq/ansible-apt-update-all-packages-on-ubuntu-debian-linux/
+- name: "[INSTALL] -  Update packages cache."
+  apt: update_cache=yes force_apt_get=yes cache_valid_time=3600
+
+- name: "[INSTALL] - Upgrade all packages to latest version."
+  apt: upgrade=dist force_apt_get=yes
+  when: upgrade_packages_first is defined
```

Comment:
What is the type of `upgrade_packages_first` ?
Looks like a feature flag (so a boolean)
But this condition will always be true even if the variable  is set to false.
It may be necessary to add a default  value  into defaults/main.yml